frontend/XCITE/SendCoins/SendDiode.qml

```diff
@@ -99,72 +100,76 @@ Controls.Diode {
                     Layout.topMargin: diodeTopMargin
                     Layout.bottomMargin: diodePadding
                     Layout.rightMargin: diodePadding
-                    Layout.maximumHeight: 504
+                    Layout.alignment: Qt.AlignTop
```

Comment:
This line should be removed since `Layout.alignment` is already set above.

Reply:
Bad diff/merge during the hassle of uploading it - will sort, thanks